faucet.config.js

```diff
@@ -1,45 +1,39 @@
+const path = require('path')
+
 const config = {
-  static: {
-    manifest: {
-      file: 'public/assets/static.json',
-      baseURI: (bundlePath, baseName) => `${process.env.BASE_URI || ''}/assets/static/${baseName}`
-    },
-    bundles: [
-      {
-        source: 'lib/images',
-        target: 'public/assets/static'
-      }
-    ]
-  },
-  sass: {
-    manifest: false,
-    assets: [
-      'public/assets/static.json'
-    ],
-    bundles: [
-      {
-        entryPoint: 'lib/styles/index.scss',
-        target: 'public/assets/stylesheets/bundle.css'
-      },
-      {
-        entryPoint: 'node_modules/normalize.css/normalize.css',
-        target: 'public/assets/stylesheets/normalize.css'
-      }
-    ]
+  manifest: {
+    file: './public/assets/static.json',
+    value: filepath => {
+      let filename = path.basename(filepath)
+      return `${process.env.BASE_URI || ''}/assets/static/${filename}`
+    }
   },
-  js: {
-    manifest: false,
-    bundles: [
-      {
-        entryPoint: 'lib/scripts/index.js',
-        target: 'public/assets/javascripts/bundle.js'
-      },
-      {
-        entryPoint: 'node_modules/document-register-element/build/document-register-element.js',
-        target: 'public/assets/javascripts/document-register-element.js'
-      }
-    ]
-  }
+  static: [
+    {
+      source: './lib/images',
+      target: './public/assets/static'
+    }
+  ],
+  sass: [
+    {
+      source: './lib/styles/index.scss',
+      target: './public/assets/stylesheets/bundle.css'
+    },
+    {
+      source: './node_modules/normalize.css/normalize.css',
```

Comment:
FYI, faucet-pipeline v1.0 (the finale release) will support non-relative references here:

```javascript
    source: 'normalize.css/normalize.css',
```

(This is currently disabled to simplify migration efforts.)